variables.tf

```diff
@@ -304,6 +315,17 @@ variable "postgres_host" {
   default     = ""
 }
 
+variable "portal_and_vitals_key_arn" {
+  description = "(Optional) ARN of the secret which contains the token used to unlock portal and vitals in Kong V3"
+  type        = string
+  default     = ""
+
+  # validation {
+  #   condition     = var.portal_and_vitals_key_arn != null ? can(startswith("arn:aws:", var.portal_and_vitals_key_arn)) : true
+  #   error_message = "Invalid format. Please provide a valid ARN."
+  # }
```

Comment:
why is this commented out? Seems like it would work if the default were `null`

Reply:
Forgot to uncomment after testing! Fixed that now but updated `null` to `""` as that's the default logic we're working with